packages/ai-jsx/package.json

```diff
@@ -125,6 +125,7 @@
     "react": "^16.8.0  || ^17.0.0 || ^18.0.0"
   },
   "dependencies": {
+    "@nick.heiner/openai-edge": "1.0.1-7",
```

Comment:
why did we need to fork openai-edge?

Reply:
Whoops – I thought I'd left a note about this somewhere: https://github.com/dan-kwiat/openai-edge/issues/6.